examples/isolated-functions/isolated_functions.bal

```diff
@@ -27,8 +27,15 @@ isolated class UniqueGreetingsStore {
     }
 }
 
+// A `final` variable of type `UniqueGreetingsStore` which
+// is a subtype of `isolated object {}`.
+// This variable can be accessed in an `isolated` function.
 final UniqueGreetingsStore uniqueGreetings = new;
 
+// An `isolated` variable that can be accessed within an
+// `isolated` function if it does so within a `lock` statement.
```

Comment:
```suggestion
// `isolated` function if it resides within a `lock` statement.
```

Reply:
The "if it does so" part is not clear. Shall we rephrase the sentence?

Reply:
Rewrote this as

```
// An `isolated` variable that can be accessed within
// a `lock` statement in an `isolated` function.
```

Reply:
+1